src/DotLiquid.Tests/Tags/StandardTagTests.cs

```diff
@@ -81,6 +81,36 @@ public void TestForWithNestedDictionary()
                 }
             } };
 
+            Helper.AssertTemplateResult("JaneMike", "{% for item in People %}{{ item.Value.First }}{%endfor%}",
+                Hash.FromDictionary(dictionary));
+        }
+
+        [Test]
+        public void TestForWithSingleLevelNestedDictionary()
+        {
+            var dictionary = new Dictionary<string, object> { {

```

Comment:
Fix the formatting here also. I assume that there are other use of nested bracketed collection initializers to refer to for consistency.